pkg/meta/model/job_args.go

```diff
@@ -1094,3 +1094,69 @@ func GetPlacementPolicyArgs(job *Job) (*PlacementPolicyArgs, error) {
 
 	return getOrDecodeArgsV2[*PlacementPolicyArgs](job)
 }
+
+// SetDefaultValueArgs is the argument for setting default value ddl.
+type SetDefaultValueArgs struct {
+	Col *ColumnInfo `json:"column_info,omitempty"`
+}
+
+func (a *SetDefaultValueArgs) fillJob(job *Job) {
+	if job.Version == JobVersion1 {
+		job.Args = []any{a.Col}
+	} else {
+		job.Args = []any{a}
+	}
+}
+
+// GetSetDefaultValueArgs get the args for setting default value ddl.
+func GetSetDefaultValueArgs(job *Job) (*SetDefaultValueArgs, error) {
+	if job.Version == JobVersion1 {
+		col := &ColumnInfo{}
+		err := job.DecodeArgs(col)
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		return &SetDefaultValueArgs{Col: col}, nil
+	}
+
+	return getOrDecodeArgsV2[*SetDefaultValueArgs](job)
+}
+
+// AddColumnArgs is the arguments for adding column ddl.
+type AddColumnArgs struct {
+	Col         *ColumnInfo         `json:"column_info,omitempty"`
+	Pos         *ast.ColumnPosition `json:"position,omitempty"`
+	Offset      int                 `json:"offset,omitempty"`
```

Comment:
Seems like `Offset` is never used. Just remove it and put zero in `AddColumnArgs`?